Unknown lifecycle phase "/home/containeruser/.m2" when run './mvnw clean package -P postgres'

Follow How use with Docker, Build or pull a dev image to help building from source:
```
./docker/benchbase/build-dev-image.sh
./docker/benchbase/run-dev-image.sh
```

then execute maven:
```
containeruser@96a38c738196:/benchbase$ ./mvnw clean package -P postgres
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------< com.oltpbenchmark:benchbase >---------------------
[INFO] Building BenchBase 2023-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.044 s
[INFO] Finished at: 2024-02-03T05:33:30Z
[INFO] ------------------------------------------------------------------------
[ERROR] Unknown lifecycle phase "/home/containeruser/.m2". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/LifecyclePhaseNotFoundException
```

To solve the issue, just `export MAVEN_CONFIG=` to empty the env var MAVEN_CONFIG

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -32,7 +32,7 @@
         "http_proxy": "${localEnv:http_proxy}",
         "https_proxy": "${localEnv:https_proxy}",
         "no_proxy": "${localEnv:no_proxy}",
-        "MAVEN_CONFIG": "${localEnv:HOME}/.m2",
+        "MAVEN_CONFIG_DIR": "${localEnv:HOME}/.m2",
         "LOCAL_WORKSPACE_FOLDER": "${localWorkspaceFolder}",
         "CONTAINER_WORKSPACE_FOLDER": "${containerWorkspaceFolder}"
     },

--- a/docker/benchbase/README.md
+++ b/docker/benchbase/README.md
@@ -75,9 +75,9 @@ docker pull benchbase.azurecr.io/benchbase-dev
 docker run -it --rm -v /path/to/src:/benchbase benchbase.azurecr.io/benchbase-dev
 ```
 
-> Optional: also reuse the local `MAVEN_CONFIG` and it's repository download cache with the following argument:
+> Optional: also reuse the local `MAVEN_CONFIG_DIR` and it's repository download cache with the following argument:
 >
-> `-v "${MAVEN_CONFIG:-$HOME/.m2}:/home/containeruser/.m2"`
+> `-v "${MAVEN_CONFIG_DIR:-$HOME/.m2}:/home/containeruser/.m2"`
 
 ```sh
 docker pull benchbase.azurecr.io/benchbase

--- a/docker/benchbase/devcontainer/Dockerfile
+++ b/docker/benchbase/devcontainer/Dockerfile
@@ -20,7 +20,7 @@ RUN groupadd --non-unique --gid ${CONTAINERUSER_GID} containergroup \
     && echo 'containeruser ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 RUN mkdir -p /benchbase/results && chown -R containeruser:containergroup /benchbase/
 USER containeruser
-ENV MAVEN_CONFIG=/home/containeruser/.m2
+ENV MAVEN_CONFIG_DIR=/home/containeruser/.m2
 WORKDIR /benchbase
 VOLUME /benchbase/results
 

--- a/docker/benchbase/devcontainer/Dockerfile
+++ b/docker/benchbase/devcontainer/Dockerfile
@@ -20,6 +20,8 @@ RUN groupadd --non-unique --gid ${CONTAINERUSER_GID} containergroup \
     && echo 'containeruser ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 RUN mkdir -p /benchbase/results && chown -R containeruser:containergroup /benchbase/
 USER containeruser
+# Unset the upstream MVN_CONFIG value - it interacts poorly with the mvnw wrapper.
+ENV MAVEN_CONFIG=
 ENV MAVEN_CONFIG_DIR=/home/containeruser/.m2
 WORKDIR /benchbase
 VOLUME /benchbase/results

--- a/docker/benchbase/run-dev-image.sh
+++ b/docker/benchbase/run-dev-image.sh
@@ -26,8 +26,8 @@ else
 fi
 
 cd "$rootdir"
-MAVEN_CONFIG="${MAVEN_CONFIG:-$HOME/.m2}"
-mkdir -p "$MAVEN_CONFIG" || true
+MAVEN_CONFIG_DIR="${MAVEN_CONFIG_DIR:-$HOME/.m2}"
+mkdir -p "$MAVEN_CONFIG_DIR" || true
 set -x
 SRC_DIR="${LOCAL_WORKSPACE_FOLDER:-$PWD}"
 docker run ${INTERACTIVE_ARGS:-} --rm \
@@ -39,5 +39,5 @@ docker run ${INTERACTIVE_ARGS:-} --rm \
     --env DO_FORMAT_CHECKS="$DO_FORMAT_CHECKS" \
     --env EXTRA_MAVEN_ARGS="${EXTRA_MAVEN_ARGS:-}" \
     --user "$CONTAINERUSER_UID:$CONTAINERUSER_GID" \
-    -v "$MAVEN_CONFIG:/home/containeruser/.m2" \
+    -v "$MAVEN_CONFIG_DIR:/home/containeruser/.m2" \
     -v "$SRC_DIR:/benchbase" benchbase-dev:latest $*

